### ISSUE TITLE
Bug on first time build

### ISSUE THREAD

**Issue by tarikmlaco** [ipselon/structor-starter-prepack#3]
first time running npm run build:
```
    ERROR in ./src/client/main.js
    Module not found: Error: Cannot resolve 'file' or 'directory' ./routes/routes.js in /Users/tarik/Desktop/structor/structor-starter-prepack/src/client
     @ ./src/client/main.js 13:14-43
```
After inspecting the ./src/client/ folder, i noticed there is no folder routes....is this a bug?

**Comment by ipselon** [comment]
This is my fault, I didn't fix readme here.

This is not a bug, routes folder will be created by Structor after you run `Export project` from the main menu in workspace. 

Try to export first, then run `npm run build`

**Comment by tarikmlaco** [comment]
Yes...i tried running the server...but the screen is blank on localhost:3000...I cannot do anything...
Same error...
Uncaught Error: Cannot find module "./routes/routes.js"

**Comment by ipselon** [comment]
Well, I just made the following steps, and I got page working:

1. start Structor from folder where structor-starter-prepack is unpacked
2. open `http://localhost:2222/structor`
3. select from main menu `Export project`
4. after successful exporting go to terminal and run from the same dir: `npm run build`
5. run `node ./server.js`
6. open in browser tab with address: `http://localhost:3000`

Did you do the same steps?
